grails-app/controllers/streama/GenericVideoController.groovy

```diff
@@ -60,8 +60,7 @@ class GenericVideoController {
             return
         }
 
-        genericVideoInstance.deleted = true
-        genericVideoInstance.save failOnError: true, flush: true
+        genericVideoInstance.delete flush:true
```

Comment:
hm, usually the `deleted` boolean is used everywhere, so instead when GenericVideo lists should exclude all the ones with deleted:true. where is there an issue here?

Reply:
the issue is the generic video is not deleted and gat a file error 
so when i changed this it's removed normally please check all them before you build a new version of streama

Reply:
Im not sure I totally understand what it is youre trying to fix

Reply:
after you generic a new video , you can't deleted it , so i make this for fix it 

I took it from previous versions and it's work for me so I make pull request